README.md

```diff
@@ -105,6 +105,7 @@ From [Node.js documentation](https://nodejs.org/api/path.html#path_windows_vs_po
 |         emulate          |                                          `boolean`                                           |   `false`   |    All    |                                                                                            Emulate remove process.<br>Print which folders and files will be removed<br>without actually removing them.<br>Ignores `log` parameter.                                                                                            |
 |   allowRootAndOutside    |                                          `boolean`                                           |   `false`   |    All    |                                                                                Allow removing of `root` directory or outside `root` directory.<br>It is kind of safe mode.<br>**Don't turn it on if you don't know<br>what you actually do!**                                                                                 |
 | readWebpackConfiguration |                                          `boolean`                                           |   `false`   |    All    |                        Change parameters based on webpack configuration.<br>Following webpack parameters are supported: `stats` (controls logging).<br>These webpack parameters have priority over the plugin parameters.<br>See webpack documentation for more - https://webpack.js.org/configuration                        |
+|        symlinks          |                                          `boolean`                                           |   `true`    |    All    |                                                                                                                  Resolve any symbolic links encountered, and recursively remove their contents as well.                                                                                                                       |
```

Comment:
Let's remove it. We will delete symlinks silently because it is default behavior of rm -rf.